AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/Mutations.dfy

```diff
@@ -597,8 +597,8 @@ module {:options "/functionSyntax:4" } Mutations {
     var encryptRes :- expect KMSKeystoreOperations.EncryptKey(
       plainTextTuple,
       HVUtils.SelectKmsEncryptionContextForHv2(terminalBKC),
-      item.EncryptionContext[Structure.KMS_FIELD],
```

Comment:
item will be overwritten with new mutation properties on `Structure.ConstructEncryptedHierarchicalKey` (line 605). So, using `item.*` won't work